### ISSUE TITLE
forced aligners comparison files

### ISSUE THREAD

**Issue by Supertyp** [simongonzalez/sydneyspeechscripts#1]
Hi Simon,
Do you by any chance still have the files of the comparison of the performance of forced aligners paper? Happy to meet. My email is still ANU email. 
All the best,
Wolfgang

**Comment by simongonzalez** [comment]
Hi Wolfgang,

So great to hear from you! Let me dig and find them. Do you want the Textgrid files?

Thank you for your interest in this. I am also keen for a chat if we can arrange the best time. I moved to Adelaide so we can talk through Zoom.



Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Wolfgang Barth ***@***.***>
Sent: Monday, July 8, 2024 10:44:16 AM
To: simongonzalez/sydneyspeechscripts ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [simongonzalez/sydneyspeechscripts] forced aligners comparison files (Issue #1)


Hi Simon,
Do you by any chance still have the files of the comparison of the performance of forced aligners paper? Happy to meet. My email is still ANU email.
All the best,
Wolfgang

**Comment by Supertyp** [comment]
Oh Wow Adelaide! Yes, you can just send me a zoom invite to wolfgang.barth@anu.edu.au whenever it works for you. I am very flexible. 
All the best,
Wolfgang

**Comment by simongonzalez** [comment]
Hi Wolfgang,

Please find attached the files for the analysis of the paper. I send the TextGrids and the main datafile.

We can meet then to talk about the content and provide any help on anything you may need.

Have a great weekend.

Best,

Simon
________________________________

**Comment by Supertyp** [comment]
Hi Simon,
I think the files didn't attach or didn't come through. Could you send them again please? Maybe put them in a zip folder. In the past I had issues with files not getting thought if looking like code. All the best, W

**Comment by simongonzalez** [comment]
Hi Wolfgang,

Sorry about that. I am able to see it and download it from my end. But let me try again with this email.
If you still have issues, please do not hesitate to let me know.

Best,

Simon
________________________________

**Comment by Supertyp** [comment]
Hi Simon,
it's still not coming through. Looks like the format is not on the list of github-supported formats. Could you just send it to my ANU email address?
All the best, W
